src/tests/suite/daos_obj_array.c

```diff
@@ -1397,6 +1397,141 @@ cond_ops(void **state)
 	print_message("all good\n");
 }
 
+
+#define NUM_IOS 256
+#define IO_SIZE 24
+#define NUM_TH	8
+
+struct test_async_arg {
+	int			thread_idx;
+	pthread_barrier_t	*barrier;
+	daos_handle_t		oh;
+	test_arg_t		*arg;
+};
+
+struct test_async_arg th_arg[NUM_TH];
+pthread_t test_tid[NUM_TH];
+
+static void *
+test_async_io(void *arg)
+{
+	struct test_async_arg	*targ = arg;
+	d_iov_t			dkey[NUM_IOS];
+	d_sg_list_t		sgls[NUM_IOS];
+	d_iov_t			sg_iovs[NUM_IOS];
+	daos_iod_t		iods[NUM_IOS];
+	daos_recx_t		recx[NUM_IOS];
+	char			akey_str[NUM_IOS][10];
+	char			dkey_str[NUM_IOS][10];
+	char			buf[NUM_IOS][IO_SIZE];
```

Comment:
with num_ios set to 256 thats a lot of stuff on a stack. does it survive without going over the limits?

Reply:
IO size is pretty small though

Reply:
In this case, it's a normal pthread rather than a ULT.  Google tells me default in 96K

Reply:
FWIW, scons install -j49 --analyze-stack="-i daos_obj_array" shows this for that function stack frame size.

50376 bytes:
    src/tests/suite/daos_obj_array.c:1416:test_async_io